Makefile

```diff
@@ -35,14 +35,20 @@ ifdef GCE_PD_CSI_DEBUG
 	GCFLAGS="all=-N -l"
 endif
 
+ARCH ?= amd64
+
 all: gce-pd-driver gce-pd-driver-windows
 gce-pd-driver: require-GCE_PD_CSI_STAGING_VERSION
 	mkdir -p bin
-	go build -mod=vendor -gcflags=$(GCFLAGS) -ldflags "-X main.version=$(STAGINGVERSION)" -o bin/${DRIVERBINARY} ./cmd/gce-pd-csi-driver/
+	GOARCH=$(ARCH) go build -mod=vendor -gcflags=$(GCFLAGS) -ldflags "-X main.version=$(STAGINGVERSION)" -o bin/${DRIVERBINARY} ./cmd/gce-pd-csi-driver/
```

Comment:
I suspect this might break the multi-arch build process because the `GOARCH` is set in the `Dockerfile`. Have you tried and make sure that `build-and-push-container-linux-arm64` still works as intended?

Reply:
It works. I think we pass the info through this arg --platform=linux/arm64

Reply:
So if you look into `Dockerfile` you can see eventually it calls this with GOARCH set https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/Dockerfile#L24

Reply:
Good call @leiyiz. @DW2022511, can we just set GOARCH= variable when we invoke the make command, similar to what is done in the Dockerfile?

Reply:
set GOARCH= when invoke the make command works.